lib/scan_beacon/ble112_device.rb

```diff
@@ -32,6 +33,12 @@ def open
       @file = nil
     end
 
+    def configure_port
+      if RUBY_PLATFORM =~ /linux/
+        system("stty -F #{@port} 115200 -brkint -icrnl -imaxbel -opost -isig -icanon -iexten -echo -echoe -echok -echoctl -echoke;min=0")
+      end
```

Comment:
mac os x doesn't need it.  It has defaults that seem to work out of the box.  Also, `stty` opens a connection and then leaves those settings in place on linux.  On mac os x, it does not - so I can't use it to change the baud rate.  I explored using `IO.popen` with some sort of `stty` command instead of opening the "file" directly, but I wasn't able to get that to work on either platform.  In any case, I would have needed platform specific code because the `stty` command operates differently on linux and mac.

In the future, I'd like to again explore using `IO.popen("stty ....") do |file| ... ` but I might need a second set of eyes to help.